### DISCUSSION TITLE
Add and remove buttons (vm batch creation) should switch positions

### DISCUSSION THREAD

**Discussion by brunomiguel** [fosshostorg/aarch64#12] [Ideas]
In the "Batch creation" section for VM creation, the **+** and **-** buttons should switch position. I think people tend to associate **+** with right and **-** with left, probably because it's being used like that for a very long time.

**Comment by natesales** [comment]
Agreed! - This should be a quick fix.

**Comment by natesales** [comment]
Fixed in https://github.com/natesales/aarch64/issues/13
